drivers/disk/ramdisk.c

```diff
@@ -33,6 +37,21 @@ static int disk_ram_access_status(struct disk_info *disk)
 
 static int disk_ram_access_init(struct disk_info *disk)
 {
+#if IS_ENABLED(CONFIG_DISK_RAM_EXTERNAL)
+	mm_reg_t virt_start;
+
+	if (!ramdisk_buf) {
+#if defined(CONFIG_MMU) || defined(CONFIG_PCIE)
```

Comment:
why can't you use `DEVICE_MMIO_IS_IN_RAM` here?

Reply:
Switched to only checking MMU config as PCIE is not relevant here